src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml

```diff
@@ -176,22 +174,14 @@ tests:
               minValue: 5
               maxValue: 21
 
-    ### Depending on the DUT specs we might prefer a arguments here
-    - label: "3a2: DUT updates its attributes"
-      cluster: "DelayCommands"
-      command: "WaitForMs"
-      arguments:
-          values:
-              - name: "ms"
-                value: 2000
```

Comment:
increase to 3000ms if still failing

Reply:
Sure i will update

Reply:
even though the delay is increased its still failing.

Reply:
point me to execution logs please